index.html

```diff
@@ -263,19 +263,43 @@ <h3 class="box3">
             <li class="lang">
                 <p>
                     FrameWorks
-                   </p>    
-                     <img class="imglang-title" src="images/arrow-side.png" alt="">
+                </p>
+                <img class="imglang-title" src="images/arrow-side.png" alt="">
             </li>
             <li class="lang-line"></li>
             <li class="lang">
                 <p>
                     Skills
-                   </p>    
-                     <img class="imglang-title" src="images/arrow-side.png" alt="">
+                </p>
+                <img class="imglang-title" src="images/arrow-side.png" alt="">
             </li>
             <li class="lang-line"></li>
         </ul>
-            
+
+    </section>
+    <section class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me </h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that
+                needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xzbodpro" method="post" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" required>
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required>
+                <textarea name="message" id="usermessage" cols="30" rows="10"
+                    placeholder="Write your message here"></textarea>
+                <div class="div-button">
+                    <button class="button">
+                        Get in touch
+                    </button>
+                </div>
```

Comment:
- [ ] Here you used a button for submitting the form. Please, consider giving the type `submit` for it.